terraform/cloudfoundry/elbs.tf

```diff
@@ -93,6 +93,57 @@ resource "aws_proxy_protocol_policy" "cf_uaa_haproxy" {
   instance_ports = ["443"]
 }
 
+resource "aws_elb" "cf_system" {
+  name                        = "${var.env}-cf-system"
+  subnets                     = ["${split(",", var.infra_subnet_ids)}"]
+  idle_timeout                = 19
+  cross_zone_load_balancing   = "true"
+  connection_draining         = true
+  connection_draining_timeout = 20
+
+  security_groups = [
+    "${aws_security_group.cf_api_elb.id}",
```

Comment:
I'm not sure this is the correct security group as this will be serving non-public things (eg tech docs and product pages)...

This might need a bit of thought...

Reply:
Upon thinking about it, I think this is fine as it is. in prod the two security groups are both open to the world, and in dev I don't think it's a problem if all system domain things are restricted to the office.